lib/legacy.js

```diff
@@ -51,10 +51,9 @@ $(function() {
 
   const LEFTARROW = 37;
   const RIGHTARROW = 39;
-  const dirs = {LEFTARROW:-1, RIGHTARROW:+1};
```

Comment:
This should have been
``` javascript
const dirs = {[LEFTARROW]:-1, [RIGHTARROW]:+1};
```

Reply:
Looking a few lines further down I see more complex case analysis and conclude this is really two distinct event handlers struggling to be one.